### PR TITLE
Populate FEATURES for 2.0.0

### PR DESCRIPTION
**Pull request by timriley**

### CODE REVIEW

**Comment by timriley** [comment]
[Update FEATURES.md](https://trello.com/c/dxiHKFN6/351-update-featuresmd)

File: FEATURES.md [hunk 1]
```diff
@@ -4,27 +4,18 @@
...
-## v2.0.0.alpha4 - 2021-12-07
-
-## v2.0.0.alpha3 - 2021-11-09
-
-## v2.0.0.alpha2 - 2021-05-04
-
-## v2.0.0.alpha1 - 2019-01-30
+## v2.0.0 - 2021-11-22
+
+- App class built around a container for managing app components; code from app/ is loaded into the app
```
Comment (jodosha) [inline FEATURES.md:9]: @timriley As first item, please write something about `app/` centrism

File: FEATURES.md [hunk 2]
```diff
@@ -4,27 +4,18 @@
...
+- Apps may have multiple Slices; code from `slices/[slice_name]/` is loaded into the corresponding slice
+- Deps mixin for auto-registering dependencies from the app or a slice
```
Comment (jodosha) [inline FEATURES.md:11]:
```suggestion
- `MyApp::Deps` mixin for auto-registering dependencies from the app or a slice
```

File: FEATURES.md [hunk 3]
```diff
@@ -4,27 +4,18 @@
...
+- Code autoloading using Zeitwerk
```
Comment (jodosha) [inline FEATURES.md:12]:
```suggestion
- Code autoloading using Zeitwerk
- Code reloading via Guard
```

**Review by jodosha — APPROVED** [review]

**Comment by jodosha** [comment]
@solnic Please check failing CI.

**Comment by solnic** [comment]
Hey sorry about the specs failure, it turned out that the log dir may or MAY NOT be created, so something's fishy there. I've worked around it in main and rebased this branch. I'll revisit this problem post-2.0 release.

File: FEATURES.md (hunk 1 above)
Comment (timriley) [reply FEATURES.md:9]: @jodosha as in, the app is always there, but the slices can be opt-in only if you want them?

Comment (timriley) [reply FEATURES.md:9]: @jodosha maybe you can have a go at writing this, since I think you have a pretty clear idea in your mind about what to write?

**Comment by timriley** [comment]
@jodosha have applied your feedback now; if you wouldn't mind sharing some words about the "app/ centrism", we can get this merged :)

File: FEATURES.md [hunk 4]
```diff
@@ -4,27 +4,19 @@
...
-## v2.0.0.alpha4 - 2021-12-07
-
-## v2.0.0.alpha3 - 2021-11-09
-
-## v2.0.0.alpha2 - 2021-05-04
-
-## v2.0.0.alpha1 - 2019-01-30
+## v2.0.0 - 2021-11-22
+
+- App class built around a container for managing app components; code from app/ is loaded into the app
```
Comment (jodosha) [inline FEATURES.md:9]:
```suggestion
- Renamed _apps_ into _slices_, and _project_ into _app_
- The core of new applications is going to be `app/`. Slices are now optional.
- App class built around a container for managing app components; code from `app/` is loaded into the app
```

**Pull request merged**